.github/fabricbot.json

```diff
@@ -1,939 +0,0 @@
-            },
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "hasLabel",
-                  "parameters": {
-                    "label": "proposal"
-                  }
-                },
-                {
-                  "name": "titleContains",
-                  "parameters": {
-                    "titlePattern": "[Proposal]"
-                  }
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isInProject",
-                  "parameters": {
-                    "projectName": "New Feature Proposals"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Proposal to Project Board",
-        "actions": [
-          {
-            "name": "addToProject",
-            "parameters": {
-              "projectName": "New Feature Proposals",
-              "columnName": "Proposal Submitted"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "elzLGcECEOVC_ZaPLpJZN",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Proposal Championed"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Champion Label",
-        "actions": [
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "champion"
-            }
-          },
-          {
-            "name": "removeLabel",
-            "parameters": {
-              "label": "new"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "g890ybM7LSjdG0Q5McEEu",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Proposal Approved"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Approved Label",
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.